src/kaboom.ts

```diff
@@ -2159,16 +2163,34 @@ function state(initState: string, stateList?: string[]): StateComp {
 		events[state][event].forEach((action) => action(...args));
 	}
 
+	function checkTransition(from: string, to: string) {
+		if (!transitions?.[from]) {
+			return
+		}
+		const available = typeof transitions[from] === "string"
+			? [transitions[from]]
+			: transitions[from] as string[];
+		if (!available.includes(to)) {
```

Comment:
Seems unnecessary to create an array here.

```suggestion
		const isAvailable = typeof transitions[from] === "string"
			? transitions[from] === to
			: transitions[from].includes(to)
		if (!isAvailable) {
```

Reply:
Absolutely

Reply:
Oh the array is used [later](https://github.com/replit/kaboom/blob/b0d1f50d68fb0f90d72bc325df5c480376bb222a/src/kaboom.ts#L2174) to print the available transition destinations

Reply:
Gotcha